agent/util-scripts/test-bin/rpmdev-vercmp

```diff
@@ -3,18 +3,27 @@
 args="${@}"
 echo "${0} ${args}" >> ${_testlog}
 
+# We are mocking the behavior of rpmdev-vercmp where the behavior of this
+# mock is directed by the 2nd arguments name.  This allows specific test
+# cases to request a certain behavior from rpmdev-vercmp.
+
 if [[ "${2}" == "not-greater-equal-1.0.0" ]]; then
+    echo "${1} <= ${2}"
     _rc=12
 elif [[ "${2}" == "greater-equal-1.0.0" || "${2}" == "greater-1.0.0" ]]; then
+    echo "${1} >= ${2}"
```

Comment:
If the caller's expectation is that the comparison is strictly "greater than", shouldn't the echo'd string be `>` rather than `>=`?

Reply:
Similar to the previous comment, a return value of `11` indicates strict `>`.